dot/simswap/util/reverse2original.py

```diff
@@ -103,22 +99,21 @@ def reverse2wholeimage(
     use_mask=False,
     use_gpu=True,
 ):
-
     target_image_list = []
     img_mask_list = []
+    device = torch.device("cuda" if use_gpu else "cpu")
     if use_mask:
-        if use_gpu:
-            smooth_mask = SoftErosion(
-                kernel_size=17, threshold=0.9, iterations=7
-            ).cuda()
-        else:
-            smooth_mask = SoftErosion(kernel_size=17, threshold=0.9, iterations=7).cpu()
+        smooth_mask = SoftErosion(kernel_size=17, threshold=0.9, iterations=7).to(
+            device
+        )
     else:
         pass
 
```

Comment:
typo in the comment: **invert** the Affine transformation matrix

Reply:
Fixed